examples/.babelrc

```diff
@@ -1,7 +1,7 @@
 {
   "plugins": [
-    "styless",
+    ["../dist", { "lessOptions": { "javascriptEnabled": true } }],
```

Comment:
I understand that it is easier to test this way, but I prefer importing `styless` to test the npm package.

Reply:
done